.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: elixir
+elixir:
+  - 1.4.5
+  - 1.5.1
```

Comment:
1.5.2

Reply:
How about we just cover the most current release then?

Reply:
sgtm